cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -426,17 +426,16 @@ CacheAllocator<CacheTrait>::allocateInternalTier(TierId tid,
                                                  uint32_t expiryTime,
                                                  bool fromBgThread,
                                                  bool evict) {
-  util::LatencyTracker tracker{stats().allocateLatency_};
-
+  util::LatencyTracker tracker{stats().allocateLatency_, ~fromBgThread};
```

Comment:
I don't understand the logic here, why `~`?

Reply:
please see the comment below